packages/react-mutation-mapper/src/util/FilterUtils.ts

```diff
@@ -74,6 +74,25 @@ export function updatePositionHighlightFilters(
     dataStore.setHighlightFilters([...defaultFilters, positionFilter]);
 }
 
+export function updatePositionRangeHighlightFilters(
+    dataStore: DataStore,
+    startPosition: number,
+    endPosition: number,
+    defaultFilters: DataFilter[] = []
+) {
+    dataStore.clearHighlightFilters();
+
+    let positionArr: number[] = [];
+    for (let i = startPosition; i <= endPosition; i++) {
+        positionArr.push(i);
+    }
+    const positionFilter = {
+        type: DataFilterType.POSITION,
+        values: positionArr,
+    };
+    dataStore.setHighlightFilters([...defaultFilters, positionFilter]);
+}
+
```

Comment:
Let's update `updatePositionHighlightFilters` so that it can accept an array of positions. Then we change this function to something like this:

```suggestion
export function updatePositionRangeHighlightFilters(
    dataStore: DataStore,
    startPosition: number,
    endPosition: number,
    defaultFilters: DataFilter[] = []
) {
    const positions: number[] = [];
    
    for (let i = startPosition; i <= endPosition; i++) {
        positions.push(i);
    }
    
    updatePositionHighlightFilters(dataStore, positions, defaultFilters);
}

```